unity-renderer/Assets/Scripts/MainScripts/DCL/Bridges/DebugBridge.cs

```diff
@@ -69,5 +64,8 @@ public void DumpRendererLockersInfo()
 
             Debug.unityLogger.logEnabled = prevLogValue;
         }
+
+        [ContextMenu("Run Performance Meter Tool for X seconds")]
```

Comment:
This might be missleading, let's replace the `X` for `10`.

This looks like an util to run the tool for Editor. We could use the `#if UNITY_EDITOR`.

Reply:
good idea, done!